tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -4,6 +4,13 @@ xcodeproj(
     name = "Single-Static-Framework-Project",
     testonly = True,
     bazel_path = "bazelisk",
+    custom_build_options_by_config = {
+        "Release": [
+            "-c",
```

Comment:
are these Bazel build options? or this can also accept generic compiler flags? From the doc I assume it means each string must be in a format of `--flag=value`

Reply:
`-c opt` is same as `--compliation_mode=opt`

Reply:
then why not just match this with debug one?

Reply:
Wanna test the case that there are multiple build options. I guess I can add another build option.

Reply:
i see! yup totally makes sense, maybe a comment or more documentation inside this line https://github.com/bazel-ios/rules_ios/pull/171/files#diff-5a136d348b6b1fd482be6ac7e567d20f52b1e808552bcede896900fc65bda48bR805 will help preserve this knowledge

Reply:
Updated